ui/src/components/MultixIdenticon.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Tooltip from "@material-ui/core/Tooltip";
+import Identicon from "@polkadot/react-identicon";
+
+interface Props {
+  value?: string,
+  theme?: any,
+  size: number,
+  className?: string;
+}
+const MultixIdenticon = ({value,theme,size,className} : Props) => {
+  
+  return (
+    <div>
+      <Tooltip
```

Comment:
This adds a constant tooltip to the icon eventually. There is no notion of "appearing on click", or being invisible after a couple seconds. You need to take this into consideration.